### PR TITLE
Sheryl doctors category feature specs and style fixes on index page 

### PR DESCRIPTION
**Pull request by sheryljohn**

### CODE REVIEW

File: spec/features/doctor_categories/create_spec.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  end
+
+  scenario 'create new doctor categories with alpha numeric letters' do
+    visit 'doctor_categories/new'
+    fill_in "Name", :with => "423434**^^^"
+    click_button "Create Doctor category"
+    expect(page).to have_content 'New Doctor Category'
+  end
+
+end
```
Comment (mathewvivek) [inline spec/features/doctor_categories/create_spec.rb]: :+1:

File: spec/features/doctor_categories/delete_spec.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+require 'spec_helper'
+require 'rails_helper'
+
+feature 'Delete Doctor Category' do
+
+  scenario 'Delete doctor category' do
```
Comment (mathewvivek) [inline spec/features/doctor_categories/delete_spec.rb]: Please don't have the same name for feature as well as scenario

File: spec/features/doctor_categories/delete_spec.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  	@doctor_categories = FactoryGirl.create_list(:doctor_category, 1)
+  	visit "doctor_categories"
+  	click_link "Destroy"
+    expect(page).to have_content 'doctor category was successfully destroyed.'
+  end
+
+end
```
Comment (mathewvivek) [inline spec/features/doctor_categories/delete_spec.rb]: :+1:

File: spec/features/doctor_categories/edit_spec.rb [hunk 4]
```diff
@@ -0,0 +1,30 @@
+require 'spec_helper'
+require 'rails_helper'
+
+feature 'Edit Doctor Category' do
+
+  scenario 'Edit doctor category' do
```
Comment (mathewvivek) [inline spec/features/doctor_categories/edit_spec.rb]: same here :)

File: spec/features/doctor_categories/edit_spec.rb [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+
+  scenario 'Edit doctor category with alpha numeric letters' do
+  	dc = FactoryGirl.create(:doctor_category)
+    visit "doctor_categories/#{dc.id}/edit"
+    fill_in "Name", :with => "45565**%$^"
+    click_button "Update Doctor category"
+    expect(page).to have_content 'Editing doctor'
+  end
+end
```
Comment (mathewvivek) [inline spec/features/doctor_categories/edit_spec.rb]: :+1:

File: spec/features/doctor_categories/index_spec.rb [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+
+feature 'Listing Doctor Categories ' do
+
+  scenario 'List doctor categories datas' do
+  	@doctor_categories = FactoryGirl.create_list(:doctor_category, 10)
+  	visit "doctor_categories"
+    expect(page).to have_content 'Listing Doctors Category'
+  end
+
+end
```
Comment (mathewvivek) [inline spec/features/doctor_categories/index_spec.rb]: :+1:

File: spec/features/doctor_categories/show_spec.rb [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+
+feature 'Show Doctor Category' do
+
+  scenario 'Show doctor category' do
+  	dc = FactoryGirl.create(:doctor_category)
+    visit "doctor_categories/#{dc.id}"
+    
+    expect(page).to have_content dc.name
+  end
+end
```
Comment (mathewvivek) [inline spec/features/doctor_categories/show_spec.rb]: :+1:

File: spec/features/doctor_categories/edit_spec.rb [hunk 8]
```diff
@@ -25,6 +25,6 @@
     visit "doctor_categories/#{dc.id}/edit"
     fill_in "doctor_category_name", :with => "45565**%$^"
     click_button "Update Doctor category"
-    expect(page).to have_content 'Editing Doctor'
+    expect(page).to have_content 'Edit Doctor Category'
```
Comment (MarlabsKochi) [inline spec/features/doctor_categories/edit_spec.rb]: @sheryljohn Good Stuff :+1:

**Pull request merged**